openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/AddDataQualityTestV1.tsx

```diff
@@ -163,7 +164,24 @@ const AddDataQualityTestV1: React.FC<AddDataQualityTestProps> = ({
       setActiveServiceStep(3);
       setTestCaseRes(testCaseResponse);
     } catch (error) {
-      showErrorToast(error as AxiosError);
+      if (
+        (error as AxiosError).response?.status === HTTP_STATUS_CODE.CONFLICT
+      ) {
+        showErrorToast(
```

Comment:
@aniketkatkar97 , Can we directly handle the conflict case in the `showErrorToast` method? like we are handling for status codes 401 and 403.

Reply:
No, Don't want to make `showErrorToast` lengthy